utils/firestore.js

```diff
@@ -780,32 +780,186 @@ const retrieveSiteNotifications = async (siteId, isParent) => {
     }
 }
 
+/**
+ * This function is used to retrieve a list of participants from the database.
```

Comment:
Descriptive but condense them to 1-2 lines or just remove it entirely.

Reply:
I took a pass and shortened all the notes. More will be removed when we complete the changeover to only array structure for `query.firstName` and `query.lastName`.